### PR TITLE
Feature/sitebuild

### PR DESCRIPTION
**Pull request by bartaroland**
ready

### CODE REVIEW

File: src/components/Architecture/Elements/Styled.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import {
+    Row,
+    Button,
+} from 'reactstrap';
+
+export const ButtonStyle = styled(Button)`
+background-color: #222A2D;
+margin-top: auto;
+margin: 0em 0em 1em 0em;
```
Comment (Roverr) [inline src/components/Architecture/Elements/Styled.jsx]:
```
margin: 25px 50px 75px 100px;
top margin is 25px
right margin is 50px
bottom margin is 75px
left margin is 100px
```
Can we go `margin-bottom: 1em;` in cases like this? Those zeros looks confusing to be honest, and the top is 0em and auto as well

File: src/components/Architecture/Architecture.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import showcase from './Elements/Assets/showcase.svg';
+import PicRow from './Elements/PicRow';
+
+export default class Architecture extends React.Component {
+    render() {
+        return (
+            <Container fluid>
+                <SubHeader title="Architecture"/>
+                <SubContent 
+                title = "Every good product starts with a fitting architecture"
```
[the remark this answers is not in the document]
Comment (bartaroland) [reply src/components/Architecture/Architecture.jsx]: :+1:

File: src/components/Architecture/Elements/Styled.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import {
+    Row,
+    Button,
+} from 'reactstrap';
+
+export const ButtonStyle = styled(Button)`
+background-color: #222A2D;
```
[the remark this answers is not in the document]
Comment (bartaroland) [reply src/components/Architecture/Elements/Styled.jsx]: :+1:

File: src/components/Architecture/Elements/Styled.jsx (hunk 1 above)
Comment (bartaroland) [reply src/components/Architecture/Elements/Styled.jsx]: :+1:

**Pull request merged**